ucm2/broadwell-rt286/HiFi.conf

```diff
@@ -49,65 +56,60 @@ SectionDevice."Headphones" {
 	]
 
 	Value {
-		Priority 200
+		PlaybackPriority 200
 		PlaybackPCM "hw:${CardId}"
-		PlaybackChannels 2
 		PlaybackMixerElem "DAC0"
-		PlaybackMasterElem "Master"
 		JackDev "rt286-jack"
 		JackControl "Headphone Jack"
-		JackHWMute "Speaker"
 	}
 }
 
 SectionDevice."Mic" {
-	Comment "Microphone"
+	Comment "Internal Microphone"
 
 	ConflictingDevice [
-		"Handset"
+		"Headset"
 	]
 
 	EnableSequence [
 		cset "name='ADC 0 Mux' 2"
+		cset "name='ADC0 Capture Switch' on"
```

Comment:
The switch should be set by the PA (CaptureMixerElem / CaptureMasterElem).

Reply:
There was a line in the `DisableSequence` to clear the switch though, which was causing it to remain off the first time that the capture device changed. Now I removed that line instead.